packages/augur.js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "augur.js",
-  "version": "6.0.1",
+  "version": "6.0.0-0",
```

Comment:
I thought this got changed to 6.0.1 ?

Reply:
The version needs to match what is required in the other modules for proper linking.

Reply:
It breaks this: https://github.com/AugurProject/augur/pull/96#issuecomment-440855225